content/finances.html

```diff
@@ -1,52 +1,85 @@
+  <br /> <br /> <!-- Newline -->
+  The average undergraduate tuition for Canadian citizens studying in Canada is around $6,693/year compared to $17,000/year for SE.
 </p>
-<!-- Graph -->
-<div class="half second">
-  <span class="graph-title">how students budgetted their money</span>
-  <div id="budgeting"></div>
-</div>
-
-<div class="vbx2"></div>
-
-<h3> Investing </h3>
-<p class="half">
-  <span class="big">
-    85% of respondents invested their money.
-  </span>
-  Index Funds and Managed Services were the most popular methods of investments, used in 60% of responses. 
-  These methods provide an easy, diversified, and low-cost way to invest, making them a useful tool to get started with investing.
-  <br /> <br /> <!-- Newline --> 
-  "With great wealth comes great responsibility"
+<p class="half second">
+  <span class="graph-title">Tuition breakdown</span>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Term</th>
+        <th>Undergrad Domestic Tuition</th>
+        <th>Co-op Fee</th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Fall 2017</td>
+        <td>7329</td>
+        <td>709</td>
+      </tr>
+      <tr>
+        <td>Winter 2018</td>
+        <td>7392</td>
+        <td>709</td>
+      </tr>
+      <tr>
+        <td>Fall 2018</td>
+        <td>7761</td>
+        <td>729</td>
+      </tr>
+      <tr>
+        <td>Spring 2019</td>
+        <td>7761</td>
+        <td>729</td>
+      </tr>
+      <tr>
+        <td>Winter 2020</td>
+        <td>6985</td>
+        <td>729</td>
+      </tr>
+      <tr>
+        <td>Fall 2020</td>
+        <td>6985</td>
+        <td>734</td>
+      </tr>
+      <tr>
+        <td>Spring 2021</td>
+        <td>6985</td>
+        <td>739</td>
+      </tr>
+      <tr>
+        <td>Winter 2022</td>
+        <td>6985</td>
+        <td>739</td>
+      </tr>
+    </tbody>
+    </table>
 </p>
-<div class="half second">
-  <span class="graph-title">How students invested their money</span>
-  <div id="investing"></div>
-</div>
-
-<div class="vbx2"></div>
-
-<h3> Family Funding </h3>
+<div class="vb"></div>
+<h3> Funding </h3>
 <p class="half">
   <span class="big">
-    80% of respondents received some funding from their family.
+    88% of respondents payed for at least half of their tutuion using money earned from coop.
   </span>
-  For Canadians, a popular source of funding is a Registered Education Savings Plan (RESP). 
-  Other methods of funding could include assisting in tuition payments, rent payments or contributions to other living expenses.
   <br /> <br /> <!-- Newline -->
-  This may suggest that the families of the majority of SE students had already planned for their children to attend post-secondary education.
+  The next largest portion of income came from respondent's families. For Canadians, this is typically from a Registered Education Savings Plan (RESP).
```

Comment:
Any idea why there is such a huge gap between the text (for basically all of the questions)? I don't remember seeing this in my section, but maybe I missed it? It looks a little odd in the screenshots to me.

Reply:
cleaned that up! Thanks for pointing it out